core/configuration/ignored_identifier.py

```diff
@@ -66,7 +66,7 @@ def set_ignored_identifier_types(
             if isinstance(item, str):
                 ignored_identifier_types.append(item)
             elif isinstance(item, IdentifierType):
-                ignored_identifier_types.append(item.name)
+                ignored_identifier_types.append(item.value)
```

Comment:
This change seems unrelated, but it was necessary when adding Proquest identifiers to the `test_opds2_importer_skips_publications_with_unsupported_identifier_types` test. Without this change, Proquest identifiers are not ignored correctly.

This function `set_ignored_identifier_types` is only used in tests in `test_odl2.py` and `test_opds2_import.py`, so the tests passing should suffice to validate the change. 

Because `item.name` and `item.value` are the same for `IdentifierType.ISBN` and `IdentifierType.URI` this wasn't noticed before, but for `IdentifierType.PROQUEST_ID` they are different, triggering this issue.